pkgs/tools/misc/btdu/default.nix

```diff
@@ -40,7 +40,8 @@ stdenv.mkDerivation rec {
     '';
 
 
-    nativeBuildInputs = [ ldc dub gdc dtools dmd ncurses ];
+    nativeBuildInputs = [ dub gdc dmd ];
```

Comment:
that's still 2 compilers: gdc and dmd. According to the d wiki ldc is better than dmd, and gdc is too old to compiler btdu.

https://wiki.dlang.org/Compilers

Reply:
I actually tested removing one of them and it didn't work.

Reply:
yeah I got to this point already haha, thx anyway! I'm now trying to figure out the reference thing u suggested I do. There isn't any sort of documentation on the script so it's been hard.